spec/models/article_spec.rb

```diff
@@ -0,0 +1,20 @@
+require "rails_helper"
+
+RSpec.describe Article, type: :model do
+  describe "valid factory" do
+    subject { build(:article) }
+
+    it { is_expected.to be_valid }
+  end
+
+  describe "validations" do
+    before { create(:article) }
+    it { is_expected.to validate_presence_of :title }
+    it { is_expected.to validate_presence_of :body }
+    it { should validate_length_of(:body).is_at_least(10).on(:create) }
```

Comment:
`should` and `is_expected` are the same, better to choose one, and I think we can omit `on(:create)`
```suggestion
    it { is_expected validate_length_of(:body).is_at_least(10) }
```